Backpack/src/main/java/net/skyscanner/backpack/calendar/view/CalendarView.kt

```diff
@@ -49,8 +49,7 @@ internal class CalendarView constructor(
       }
     }
 
-  var listener: OnYearChangedListener? = null
-  var scrollListener: OnScrollListener? = null
+  var calendarScrollListeners = mutableSetOf<BpkCalendarScrollListener>()
```

Comment:
Can you please make this private and add `add` and `remove` listener functions?

Reply:
I updated the PR

Reply:
This seems to be still the same, can we make this private and add functions to add and remove listeners?

Reply:
Sorry, forgot to push!